PolyEngine/Core/Src/Utils/FileIO.hpp

```diff
@@ -18,6 +17,7 @@ namespace Poly {
 #endif
 
 	//------------------------------------------------------------------------------
+	SILENCE_MSVC_WARNING(4275, "Exporting stl may cause incompatibility. We use same CRT so it's ok.")
```

Comment:
Is it possible to remove this silencing?

Reply:
For now it is not possible to remove this silencing without a refactor.